labgrid/target.py

```diff
@@ -481,3 +481,11 @@ def cleanup(self):
         self.deactivate_all_drivers()
         for res in reversed(self.resources):
             self.deactivate(res)
+
+    def get_target_features(self):
+        """Helper to retrieve all features of this target"""
+        return self.env.config.get_target_features(self.name)
```

Comment:
Hmm, maybe it would make more sense to pass the features and options to make_target() and store them as a copy in each target? That would avoid following the path upwards from target to env to config...